OPT_mission.Tanoa/beam/functions/fnc_addbeam.sqf

```diff
@@ -0,0 +1,66 @@
+/**
+* Author: form
+* update beam positions
+*
+* Arguments:
+* None
+*
+* Return Value:
+* None
+*
+* Example:
+* [] call fnc_addBeam.sqf;
+*
+*/
+#include "script_component.hpp"
+
+#define ADDBEAM_IDC 1413
+
+disableSerialization;
+
+private _dialog = uiNamespace getVariable [QGVAR(AddBeamDialog) , displayNull];
+if (_dialog isEqualTo displayNull) exitWith {};
```

Comment:
In dem `exitWith` sollte via `ERROR_LOG` ein Fehler geloggt werden

Reply:
Meinst Du das ERROR_MSG Makro von CBA?

Reply:
Nee ich meinte das `ERROR_LOG` aus der OPT Servermod. Aber das hier ist ja die Mission :man_facepalming: 

Das wichtige hierbei ist, dass du den Fehler nicht einfach stillschweigend abfederst, sondern in irgendeiner Form was ins Log schreibst, damit man später nachvollziehen kann, wann und wo was schiefgelaufen ist. Welche Funktion du dabei zum Loggen nutzt ist im Endeffekt egal :)